.env

```diff
@@ -0,0 +1,14 @@
+
+PORT = 3000
+
+DOCTOR_ACCESS_TOKEN_SECRET = '970d8bf7e7e3627f066087ae3fdee48c48e086c4912780e52b3a6b5b7792fc08d2fb0714853b3b5e7cd56cd489568818378f6f05d69b9fe519ebbd970ec5f3b7'
+
+PATIENT_ACCESS_TOKEN_SECRET ='14992ecad469e42b4c4299002a09e8511e6138e5c855f5e1f3e0e690292cec6d7ca2bb4efa44420c2dc8085d69254d86fce6855cda0eac8824cf89ba6b02a616'
+
+DOCTOR_REFRESH_TOKEN_SECRET = 'b9d7f333cb181f0e9f2da739a29bd0625649952d7b0f11ef5736961f801766179e4420a4fbabfb87c4cb7e0b9cffe494af6e8f0687be89a9f20c952d9d3075d6'
+
+PATIENT_REFRESH_TOKEN_SECRET = 'b6eedf5700bc6c2c59ac9920b21f5cb699bdf61bfd0b12638bd0477b628c07eca146f4da456416f46e6a5a3d22b07d0bb292444aeca0ef0f4f9909b6ec966fc6'
+
+
+
+
```

Comment:
`.env` file should never be uploaded to git. Instead always create a `.env-sample` and add the keys to it, and create a `readme.md` file and write down what are the keys used for and how can we get them.

Reply:
Fixed in here, removed .env file in [PR2](https://github.com/Sankalp-Sachapara/HospitalPractice/pull/2/files/893ce87fe4fe8a5be2c037597a3a6cfbdffc56ac#diff-c4368a972a7fa60d9c4e333cebf68cdb9a67acb810451125c02e3b7eb2594e3dR1-R11)

Reply:
Added readme.md [Readme.md](https://github.com/Sankalp-Sachapara/HospitalPractice/pull/2/files/893ce87fe4fe8a5be2c037597a3a6cfbdffc56ac#r1045075087)

Reply:
👍